src/geo_index.c

```diff
@@ -213,7 +214,7 @@ int GeoFilter_Validate(const GeoFilter *gf, QueryError *status) {
   }
 
   // validate lat/lon
-  if (gf->lat > 90 || gf->lat < -90 || gf->lon > 180 || gf->lon < -180) {
+  if (gf->lat > GEO_LAT_MAX || gf->lat < GEO_LAT_MIN || gf->lon > GEO_LONG_MAX || gf->lon < GEO_LONG_MIN) {
```

Comment:
@ashtul Need to discuss if this is considered a breaking change (zero results Vs. error)